crates/cxx-qt-lib/src/gui/qcolor.rs

```diff
@@ -81,6 +475,24 @@ impl Default for QColor {
     }
 }
 
+impl From<&str> for QColor {
+    fn from(value: &str) -> Self {
+        ffi::qcolor_init_from_rust_string(value)
+    }
+}
+
+impl From<&String> for QColor {
+    fn from(value: &String) -> Self {
+        ffi::qcolor_init_from_rust_string(value)
+    }
+}
+
+impl From<&ffi::QString> for QColor {
+    fn from(value: &ffi::QString) -> Self {
+        ffi::qcolor_init_from_qstring(value)
+    }
+}
+
```

Comment:
These should probably all be TryFrom implementations and use an invalid check

Reply:
done